packages/tpetra/core/src/Tpetra_CrsGraph_def.hpp

```diff
@@ -4735,6 +4752,70 @@ namespace Tpetra {
     } // if my process doesn't have local constants
   }
 
+  template <class LocalOrdinal, class GlobalOrdinal, class Node, const bool classic>
+  void
+  CrsGraph<LocalOrdinal, GlobalOrdinal, Node, classic>::
+  computeTriangularProperties ()
```

Comment:
I decided not to make it a free function as it still sets some CrsGraph members.